queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SSELECT * FROM animals WHERE name LIKE '%mon';
```

Comment:
- [ ] You have two `S` in the **SSELECT** in line 3 which is making your query not work 😨 Kindly remove one `S` to make your query work without error 👍